restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -436,7 +437,7 @@
 	"upgradeNeedsUpgradeLabel3": "Actualice",
 	"upgradeNeedsUpgradeLabel4": "o",
 	"upgradeNeedsUpgradeLabel5": "visualice los proyectos disponibles",
-	"upgradeAlreadyUpgradedLabel1": "Proyecto",
+	"upgradeAlreadyUpgradedLabel1": "El proyecto",
 	"upgradeAlreadyUpgradedLabel2": "ya esta actualizado.",
 	"upgradeAlreadyUpgradedLabel3": "Abrir",
 	"upgradeAlreadyUpgradedLabel4": "o",

```

Comment:
Error found in line 454. Change for:
"ussdCollectStepCollectButton": "Capturar la respuesta",